tasks/create-github-release/README.md

```diff
@@ -14,6 +14,9 @@ a `release` dir.
 | release_version | The version string to use creating the release | No | - |
 | content_directory | The directory inside the workspace to find files for release | No | - |
 
+## Changes since 1.0.1
```

Comment:
Can you change this to `Changes in 1.0.2` please? We changed our changelog style